### DISCUSSION TITLE
七个孙子 一个一个送hhh

### DISCUSSION THREAD

**Discussion by keyuanya** [share121/inter-knot#5159] [崩坏：星穹铁道]
![StarRail_Image_1725987129](https://github.com/user-attachments/assets/268fa0ab-ae39-497d-9b6d-8a1d3a8526ac)把劫匪都搞烦了
![Screenshot_20240911_005742_com miHoYo hkrpg](https://github.com/user-attachments/assets/4f3f1ef2-4ffd-4ede-9584-60c296d84c2e)

**Comment by zjcbhj** [comment]
葫芦娃
